dotcom-rendering/src/web/components/Card/components/LI.tsx

```diff
@@ -52,6 +52,10 @@ const decideSize = (percentage?: CardPercentageType, stretch?: boolean) => {
 	if (percentage) {
 		sizeStyle = css`
 			flex-basis: ${percentage};
+			${stretch &&
+			css`
+				flex-grow: 1;
+			`}
```

Comment:
Here I added the ability to enable 'stretch={true}` while also setting a percentage. This was needed to achieve some of the layouts required

Reply:
(this seems to have had some side effects else where which I'm not looking into)